src/openzaak/components/documenten/tests/test_filters.py

```diff
@@ -1,18 +1,82 @@
 # SPDX-License-Identifier: EUPL-1.2
 # Copyright (C) 2019 - 2020 Dimpact
+from django.contrib.sites.models import Site
+from django.test import override_settings
+
 from rest_framework import status
 from rest_framework.test import APITestCase
 from vng_api_common.tests import get_validation_errors, reverse
 
+from openzaak.components.zaken.tests.factories import (
+    ZaakFactory,
+    ZaakInformatieObjectFactory,
+)
 from openzaak.tests.utils import JWTAuthMixin
 
-from ..models import Gebruiksrechten, ObjectInformatieObject
+from ..models import (
+    EnkelvoudigInformatieObject,
+    Gebruiksrechten,
+    ObjectInformatieObject,
+)
 from .factories import (
     EnkelvoudigInformatieObjectCanonicalFactory,
+    EnkelvoudigInformatieObjectFactory,
     GebruiksrechtenFactory,
 )
 
 
+class EnkelvoudigInformatieObjectFilterTests(JWTAuthMixin, APITestCase):
+    heeft_alle_autorisaties = True
+
+    def test_locked_filter(self):
+        locked_canonical = EnkelvoudigInformatieObjectCanonicalFactory(lock="locked")
+        EnkelvoudigInformatieObjectFactory(canonical=locked_canonical)
+        EnkelvoudigInformatieObjectFactory()
+
+        with self.subTest("locked_objects"):
+            response = self.client.get(
+                reverse(EnkelvoudigInformatieObject), {"locked": True}
+            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            data = response.json()
+            self.assertEqual(data["count"], 1)
+            self.assertEqual(data["results"][0]["locked"], True)
+
+        with self.subTest("unlocked_objects"):
+            response = self.client.get(
+                reverse(EnkelvoudigInformatieObject), {"locked": False}
+            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            data = response.json()
+            self.assertEqual(data["count"], 1)
+            self.assertEqual(data["results"][0]["locked"], False)
+
+    @override_settings(
+        NOTIFICATIONS_DISABLED=True, ALLOWED_HOSTS=["testserver", "testserver.com"],
+    )
+    def test_zaak_filter(self):
```

Comment:
Since the zaak filter is a `NestedFkOrUrlFieldFilter`, could you also add a test to check if filtering works if the zaak is external?